src/Perpetuum/Accounting/AccountManager.cs

```diff
@@ -132,7 +139,7 @@ public int GetLockedEpByAccount(Account account)
 
         private static double GetBoostMultiplier(double boostFactor)
         {
-            return (BOOSTMULTIPLIERMAX - 1) * boostFactor;
+            return (BOOSTMULTIPLIERMAX + EPBonusBoost - 1) * boostFactor;
```

Comment:
This is the right place to add the bonus, but it makes the logging incorrect. On line 366 the boost factor is calculated using the old linear scaling algorithm, but this doesn't take into account the bonus that's just been added.

Reply:
Maybe I am misunderstanding, but boostingfactor tracks only along the curve of ep-on-account over the target, (the inverse of that) clamped to [0.-1.].  The multiplier is applied after, while assumed to be constant, could be anything with the new event.  But it can still be computed from the logging w/ the raw and boosted points.
To me they are separate, and didn't think to alter the boostingFactor to compensate for an EP Bonus Event, which to keep it simple, pushes the Multiplier up.

Is there a dependency on the epForActivityLog that I am missing?

Reply:
![image](https://user-images.githubusercontent.com/3584566/44915710-93393900-ad2b-11e8-82cb-e3ed8f326934.png)

Not exactly. Line 367 calls GetBoostedExtensionPoints, this in turn calls GetEPCollected and AddEPWithBoosting. AddEPWithBoosting gets the EP Boosting factor and then re-calculated this with the multiplier to give you an overall EP gain. This is returned as the "real EP gain" and is populated into the boostedPoints variable. Now the boost factor contains the *non* boost multiplied factor (the old scaling number) but the boosted points value contains the multiplied EP. If you were to try and back calculated the EP from the log, this number would be inconsistent with what's present in the log. When the LogEpForActivity call audits this into the database, the rawPoints and actualPoints will match what were given, however, the boost factor will not. Whilst you can back-calculate this, it's not evident at a glance why the numbers wouldn't match (think new non-technical support agent)

So in order to keep the numbers consistent either;
a) We need to record an extra value in the log indicating the boost multiplication value
b) We need to pass in the correct boost factor to the audit log

So just to be clear, I'm not saying there's a functionality problem with the boost factor and how it's applied, however, the audit trail will be inconsistent and/or confusing because this information isn't being recorded.